src/main/java/com/jfrog/ide/idea/scan/ScanManager.java

```diff
@@ -285,6 +285,30 @@ void runInspections() {
         return collectAllScopes((DependencyTree) getScanResults().getRoot());
     }
 
+    /**
+     * Return true if the scan results contain any issues.
+     *
+     * @return true if the scan results contain any issues.
+     */
+    public boolean isContainIssues() {
+        if (getScanResults() == null) {
```

Comment:
The code in this method can be simplified into a 1 liner conditional return.
The same goes for the ```isContainViolatedLicenses``` method below.